packages/core/src/build-package.ts

```diff
@@ -93,13 +94,17 @@ const build = async (config: EnhancedConfig, packageName: string) => {
     packageRoot: config.root,
   });
 
-  const packageDiffs = await validatePackageJson(config.root, entries);
-
-  if (packageDiffs.length) {
-    logger.errorWithExitCode(
-      renderPackageJsonValidationError(packageName, packageDiffs),
-    );
-    return;
+  if (config.fix) {
+    await fix(config);
```

Comment:
Would be nice to get the output of what fixed by using `fixPackageJson` from `./utils/setup-package-json`.

Reply:
I replaced `fixPackageJson` with `fix` because the latter has output:

```
$ ///node_modules/.bin/crackle package --fix
No crackle.config.ts file found, using default configuration.
🛠  Building @crackle-fixtures/library-with-docs...
✅ Successfully built @crackle-fixtures/library-with-docs!

Fixed package.json for:
  @crackle-fixtures/library-with-docs
    - "main" was changed to dist/index.cjs (previously dist/index.js)
    - "module" was changed to dist/index.mjs
    - "files" updated with: /dist
    - "exports" key updated

```

`fixPackageJson` only returns the diffs.

Reply:
Going back to fixPackageJson as discussed

Reply:
I've slightly tweaked the output so now it looks like this:

```sh
$ ///node_modules/.bin/crackle package --fix
No crackle.config.ts file found, using default configuration.

Fixed package.json for:
  @crackle-fixtures/library-with-docs
    - "main" was changed to dist/index.cjs (previously dist/index.js)
    - "module" was changed to dist/index.mjs
    - "files" updated with: /dist
    - "exports" key updated

🛠  Building @crackle-fixtures/library-with-docs…
✅ Successfully built @crackle-fixtures/library-with-docs!
```